Highlight testing guide FAQ section (#2315)
w3f/Grants-Program: applications/application-template.md

| -----: | ----------- | ------------- |

> | **0a.** | License | Apache 2.0 / GPLv3 / MIT / Unlicense |

Reviewer: ```suggestion
| **0a.** | License | Apache 2.0 / GPLv3 / MIT / Unlicense. See our [delivery guidelines](https://grants.web3.foundation/docs/Support%20Docs/milestone-deliverables-guidelines#license) for details. |
```

> | **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |

Reviewer: ```suggestion
| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. See our [delivery guidelines](https://grants.web3.foundation/docs/Support%20Docs/milestone-deliverables-guidelines#documentation) for details. |
```

> | **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. Please check our requirements for testing and testing guide in the [FAQ](https://grants.web3.foundation/docs/Support%20Docs/milestone-deliverables-guidelines#testing-guide) |

Reviewer: ```suggestion
| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. See our [delivery guidelines](https://grants.web3.foundation/docs/Support%20Docs/milestone-deliverables-guidelines#testing-guide) for details. |
```
